.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Can we put this in the `kcd.settings.json`? I don't like surprising people with settings, but they can copy them if they want them.

Reply:
Oh yes, I think that's a good call

Reply:
Confirmed they have been copied over and the un-suffixed `settings.json` has been removed